packages/expo-image/src/web/AnimationManager.tsx

```diff
@@ -185,21 +189,23 @@ export default function AnimationManager({
 
   return (
     <>
-      {[...nodes].map((n) => (
-        <div className={animation?.containerClass} key={n.animationKey}>
-          {wrapNodeWithCallbacks(n)(
-            {
-              in: animation?.animateInClass,
-              out: animation?.animateOutClass,
-              mounted: animation?.startingClass,
-            }[n.status],
-            {
-              transitionDuration: `${animation?.duration || 0}ms`,
-              transitionTimingFunction: animation?.timingFunction || 'linear',
-            }
-          )}
-        </div>
-      ))}
+      {[...nodes]
+        .filter((n) => n.status !== 'errored')
+        .map((n) => (
+          <div className={animation?.containerClass} key={n.animationKey}>
+            {wrapNodeWithCallbacks(n)(
+              {
+                in: animation?.animateInClass,
+                out: animation?.animateOutClass,
+                mounted: animation?.startingClass,
+              }[n.status],
+              {
+                transitionDuration: `${animation?.duration || 0}ms`,
+                transitionTimingFunction: animation?.timingFunction || 'linear',
+              }
+            )}
+          </div>
+        ))}
```

Comment:
It's becoming hard to read, can you move this out of the return statement or to a separate component?